External/FEXCore/Source/Interface/Core/GdbServer.h

```diff
@@ -77,6 +77,7 @@ class GdbServer {
     std::string MemoryMapString{};
     std::string OSDataString{};
 
+    std::array<bool, SignalDelegator::MAX_SIGNALS + 1> PassSignals{};
```

Comment:
What is the extra slot used for? (I.e. why not just `MAX_SIGNALS` elements?)

Reply:
This is a pattern used in our signal delegation at the moment.
I created an issue to get this resolved.
This is due to signals being from [1,64] inclusive with 0 being special cased.

Happened since I wasn't expecting 0 to be special cased and now their is a maintenance burden. Created a issue so we can track this and get it resolved after this.